test/onnx/test_pytorch_onnx_caffe2_quantized.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import print_function
+
+import numpy as np
+import unittest
+import torch.onnx
+import io
+
+import onnx
+import caffe2.python.onnx.backend as c2
+
+class TestQuantizedOps(unittest.TestCase):
+    def generic_test(self, model, sample_inputs, input_names=None):
+        pt_inputs = tuple(torch.from_numpy(x) for x in sample_inputs)
+        model.qconfig = torch.quantization.default_qconfig
+        q_model = torch.quantization.prepare(model, inplace=False)
+        q_model = torch.quantization.convert(q_model, inplace=False)
+        pytorch_res = q_model(*pt_inputs)
+        f = io.BytesIO()
+        torch.onnx.export(q_model, pt_inputs, f, input_names=input_names, operator_export_type=torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK)
+        f.seek(0)
+        onnx_model = onnx.load(f)
+        caffe_res = c2.run_model(onnx_model, dict(zip(input_names, sample_inputs)))[0]
+        np.testing.assert_almost_equal(pytorch_res.numpy(), caffe_res, decimal=3)
+
+    def generic_unary_test(self, op):
+        class QModule(torch.nn.Module):
+            def __init__(self, op):
+                super(QModule, self).__init__()
+                self.quant1 = torch.quantization.QuantStub()
+                self.op = op
+                self.dequant = torch.quantization.DeQuantStub()
+
+            def forward(self, x):
+                res = self.op(self.quant1(x))
+                return self.dequant(res)
+
+        x = np.random.random((1, 2)).astype("float32")
+        self.generic_test(QModule(op), (x,), input_names=["x"])
+
+    def test_quantized_add(self):
+        class QAddModule(torch.nn.Module):
+            def __init__(self):
+                super(QAddModule, self).__init__()
+                self.quant1 = torch.quantization.QuantStub()
+                self.quant2 = torch.quantization.QuantStub()
+                self.dequant = torch.quantization.DeQuantStub()
+
+            def forward(self, x, y):
+                res = torch.ops.quantized.add(self.quant1(x), self.quant2(y), 1.0, 0)
+                return self.dequant(res)
+
+        x = np.random.random(2).astype("float32")
+        y = np.random.random(2).astype("float32")
+        self.generic_test(QAddModule(), (x, y), input_names=["x", "y"])
+
+    def test_quantized_relu(self):
+        self.generic_unary_test(torch.nn.ReLU())
```

Comment:
Nit: also add test to make sure regular relu can be correctly handled as well?

Reply:
`test_linear` in test_pytorch_onnx_caffe2.py already covers regular relu.